frontend/src/lib/components/EditableField/EditableField.tsx

```diff
@@ -65,16 +66,30 @@ export function EditableField({
     saveButtonText = 'Save',
     notice,
 }: EditableFieldProps): JSX.Element {
-    const [localIsEditing, setLocalIsEditing] = useState(false)
+    const [localIsEditing, setLocalIsEditing] = useState(mode === 'edit')
     const [localTentativeValue, setLocalTentativeValue] = useState(value)
+    const inputRef = useRef<HTMLInputElement | HTMLTextAreaElement>()
+    const previousIsEditing = useRef<boolean>()
```

Comment:
Why does this need to be a ref?

Reply:
Refs are typical when you want to hold a previous value, because we don't want to re-render when setting this